WIP bump up apps version

kopax opened the pull request:
For info it also fix group deployment : 

- https://jira.mesosphere.com/browse/MARATHON-7433
- Install warning from neta ([see](https://github.com/eBayClassifiedsGroup/PanteraS/pull/274/commits/c1aa9db3e2a4c48703185368bbd3ddd3eb7148d9))

sielaq commented on infrastructure/Dockerfile:
```diff
@@ -203,6 +205,9 @@ RUN apt-get update \
 
+RUN [[ -e "/sys/kernel/mm/ksm/run" ]] \
+    && echo 1 >/sys/kernel/mm/ksm/run \
+    && echo 1000 >/sys/kernel/mm/ksm/sleep_millisecs
```
not sure if de-duplication will work during creation of the container
/sys/ is created dynamically - must be set in run time mode

sielaq commented on infrastructure/Dockerfile:
```diff
@@ -40,24 +40,26 @@ RUN apt-get update \
-ENV CONSUL_APP_VERSION            0.8.5
-ENV CONSUL_TEMPLATE_APP_VERSION   0.18.2
+ENV CONSUL_APP_VERSION            0.9.3
+ENV CONSUL_TEMPLATE_APP_VERSION   0.19.3
```
please dont raise the one - consul template will not work
seems the configuration has changed recently.

sielaq commented:
build fails during `RUN [[ -e "/sys/kernel/mm/ksm/run" ]] ...`

kopax commented:
Yes I have removed that and added a `netdata.sh` in profile
Please wait for the review has I am still trying to find out why marathon is failing with:
> [scallop] Error: Unknown option 'ldap_version'

kopax commented:
I've asked at #mesos and :
```
<dka> I have upgraded marathon from 1.4.5 to 1.4.8 . I can't start marathon anymore: "[scallop] Error: Unknown option 'ldap_version'" 
<mslackbot> [byron_miller] maybe you have this installed and its now failing? https://github.com/ContainX/marathon-ldap
<mslackbot> [byron_miller] unwind that and it may startup? thats the only ldap thingy i know of beyond whats built into DCOS ¯\_(ツ)_/****
```

I've tried to remove the plugin and build up the image again but I get the same error. This is weird, I'll try again later. Closing for now.
